src/types/hermesProfileInterfaces.ts

```diff
@@ -0,0 +1,42 @@
+import { SharedEventProperties } from './chromeEventInterfaces';
+
+/**
+ * Each item in the stackFrames object of the hermes profile
+ */
+export interface HermesStackFrame {
+  line: string;
+  column: string;
+  funcLine: string;
+  funcColumn: string;
+  name: string;
+  category: string;
+  /**
+   * A parent function may or may not exist
+   */
+  parent?: number;
+}
+/**
+ * Each item in the samples array of the hermes profile
+ */
+export interface HermesSample {
+  cpu: string;
+  name: string;
+  ts: string;
+  pid: number;
+  tid: string;
+  weight: string;
+  /**
+   * Will refer to an element in the stackFrames object of the Hermes Profile
+   */
+  sf: number;
+  stackFrameData?: HermesStackFrame;
+}
+
+/**
+ * Hermes Profile Interface
+ */
+export interface HermesCPUProfile {
```

Comment:
Interface for the overall profile, consisting of the three properties as mentioned in #16's description